README.md

```diff
@@ -73,9 +73,45 @@ Watson services are migrating to token-based Identity and Access Management (IAM
 ### Getting credentials
 To find out which authentication to use, view the service credentials. You find the service credentials for authentication the same way for all Watson services:
 
-1. Go to the IBM Cloud [Dashboard][watson-dashboard] page.
-1. Either click an existing Watson service instance or click [**Create resource > AI**](https://console.bluemix.net/catalog/?category=ai) and create a service instance.
-1. Copy the `url` and either `apikey` or `username` and `password`. Click **Show** if the credentials are masked.
+1. Go to the IBM Cloud [Dashboard](https://cloud.ibm.com/) page.
+1. Either click an existing Watson service instance in your [resource list](https://cloud.ibm.com/resources) or click [**Create resource > AI**](https://cloud.ibm.com/catalog?category=ai) and create a service instance.
+1. Click on the **Manage** item in the left nav bar of your service instance.
+
+ On this page, you should be able to see your credentials for accessing your service instance.
+
+ ### Supplying credentials
+
+ There are two ways to supply the credentials you found above to the SDK for authentication.
+
+ #### Credential file (easier!)
+
+ With a credential file, you just need to put the file in the right place and the SDK will do the work of parsing and authenticating. You can get this file by clicking the **Download** button for the credentials in the **Manage** tab of your service instance.
+
+ The file downloaded will be called `ibm-credentials.env`. This is the name the SDK will search for and **must** be preserved unless you want to configure the file path (more on that later). The SDK will look for your `ibm-credentials.env` file in the following places (in order):
+
+ - Your system's home directory
+- The top-level directory of the project you're using the SDK in
+
+ As long as you set that up correctly, you don't have to worry about setting any authentication options in your code. So, for example, if you created and downloaded the credential file for your Discovery instance, you just need to do the following:
+
+ ```python
```

Comment:
ruby?

Reply:
oops, you can see where I got the instructions from 😮